server/utils/matchUtils.test.ts

```diff
@@ -346,4 +353,72 @@ describe('matchUtils', () => {
       ])
     })
   })
+
+  describe('spaceBookingSummaryCardRows', () => {
+    it('should call the correct row functions', () => {
+      const premisesName = 'Hope House'
+      const gender = 'male'
+      const apType = 'pipe'
+      const dates = {
+        startDate: '2022-01-01',
+        endDate: '2022-01-15',
+        placementLength: 2,
+      }
+      const essentialCharacteristics = ['hasTactileFlooring'] as Array<PlacementCriteria>
```

Comment:
We could type this instead of casting it, same with the desirable criteria below